pages/linux/zenity.md

```diff
@@ -0,0 +1,23 @@
+# zenity
+
+> Display dialogs from the command line/shell scripts, and return user-inserted values or 1 if error.
+
+- Display the default question dialog:
+
+`zenity --quesion`
+
+- Display an info dialog displaying the text "Hello!":
+
+`zenity --info --text="Hello!"`
```

Comment:
Could you wrap `Hello!` in `{{` and `}}`, please? That's our custom syntax that denotes that you can change a value, and is understood by tldr-pages clients.

Reply:
Hello, I didn't notice the special syntax to highlight keywords. I'll apply these changes tomorrow in the morning.